github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPRLabelExistsEvent/config.groovy

```diff
@@ -2,7 +2,7 @@ package org.jenkinsci.plugins.github.pullrequest.events.impl.GitHubPRLabelExists
 
 def f = namespace(lib.FormTagLib);
 
-f.entry(field: "skip", title: "Skip PR?") {
+f.entry(field: "skip", title: "Skip pull requests with label(s)?") {
```

Comment:
We had great holy war with @alina-karpovich about this event. It my even that i used in two real cases: 
- during testing plugin
- with labels manipulations, trigger on existing, but remove it later. 
- it covers all available labels functionality
I have infinite builds with git polling, but nobody fixing it. So i would like to keep it.